javascript.html.markdown

```diff
@@ -603,31 +603,35 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement defined will be the return statement, hence the `return` keyword is not required. 
 
-const isEven = (number) => {
-    return number % 2 === 0;
-};
-
-isEven(7); // false
+const isOdd = number => number % 2 !== 0;
 
 // The "equivalent" of this function in the traditional syntax would look like this:
 
-function isEven(number) {
-    return number % 2 === 0;
+function isOdd(number) {
+    return number % 2 !== 0;
 };
 
-// I put the word "equivalent" in double quotes because a function defined
-// using the lambda syntax cannnot be called before the definition.
+// However if a function has no arguments, a slight change of adding the parentheses in it's place is required:
+const helloWorld = () => {
+    console.log("Hello World")
+}
+
+// Also, I put the word "equivalent" in double quotes because a function defined
```

Comment:
i think the sudden personal tone is off-putting. i understand it's from a previous commit, but are there any ways you can think of rephrasing it?